lib/Mail/DMARC/Report/Store/SQL.pm

```diff
@@ -440,7 +440,7 @@ sub row_exists {
     my ($self, $rid, $rec ) = @_;
 
     if ( ! defined $rec->{row}{count} ) {
-        carp "\tnew record";
+        print "new record\n" if $self->verbose;
```

Comment:
I forget the context of this message, and notice you removed the leading tab. Do you have some output handy, showing before and after?